tasks.py

```diff
@@ -31,7 +31,7 @@ def is_truthy(arg):
 TOOL_CONFIG = PYPROJECT_CONFIG["tool"]["poetry"]
 
 # Can be set to a separate Python version to be used for launching or building image
-PYTHON_VER = os.getenv("PYTHON_VER", "3.9")
+PYTHON_VER = os.getenv("PYTHON_VER", "3.8.10")
```

Comment:
Whats this for?

Reply:
My bad. That was me messing with it to test with the pipeline. Reverted